src/components/SmartBadge/SmartBadge.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import classNames from 'classnames'
+import IconBadge from '../IconBadge'
+import { Props } from './SmartBadge.types'
+import { SmartIcon } from '../SmartIcon/SmartIcon'
+import './SmartBadge.css'
+
+const SmartBadge = ({ clickable = true, i18n }: Props) => {
+  const { title = 'Smart' } = i18n || {}
+
+  return (
+    <IconBadge className={classNames('SmartBadge', { clickable })} text={title}>
+      <SmartIcon />
```

Comment:
Just out of curiosity, is it possible to add the smart icon to the set of icons the `IconBadge` supports?

Reply:
good idea, updated!